docs/guides/oracle/initialization/script_source.md

```
@@ -0,0 +1,151 @@
+
+- Now, install KubeDB cli on your workstation and KubeDB operator in your cluster following the steps [here](/docs/setup/README.md).
+
+- To keep things isolated, this tutorial uses a separate namespace called `demo` throughout this tutorial.
+
+```bash
+$ kubectl create ns demo
+namespace/demo created
+```
+
+> Note: YAML files used in this tutorial are stored in [docs/examples/oracle/initialization](https://github.com/kubedb/docs/tree/{{< param "info.version" >}}/docs/examples/oracle/initialization) folder in GitHub repository [kubedb/docs](https://github.com/kubedb/docs).
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)